docs/proposals/pod-resource-management.md

```diff
@@ -0,0 +1,394 @@
+/Bu/PodUID/m.limit = Pod memory limit
+```
+
+`Note: For the Burstable Qos when limits are not specified for any one of the containers, the Pod limit defaults to the node resource allocatable quantity.`
+
+Example:
+We have two pods Pod3 and Pod4 having Pod Spec given below:
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod3
+spec:
+    containers:
+        name: foo
+            resources:
+                limits:
+                    cpu: 50m
+                    memory: 2Gi
+                requests:
+                    cpu: 20m
+                    memory: 1Gi
+        name: bar
+            resources:
+                limits:
+                    cpu: 100m
+                    memory: 1Gi
+```
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod4
+spec:
+    containers:
+        name: foo
+            resources:
+                limits:
+                    cpu: 20m
+                    memory: 2Gi
+                requests:
+                    cpu: 10m
+                    memory: 1Gi  
+```
+
+Pod3 and Pod4 are both classified as Burstable and are hence nested under the Bu cgroup
+
+```
+/Bu/cpu.shares = 30m  
+/Bu/Pod3/cpu.quota = 150m  
+/Bu/Pod3/cpu.shares = 20m  
+/Bu/Pod4/cpu.quota = 20m  
+/Bu/Pod4/cpu.shares = 10m  
+/Bu/m.limit_in_bytes = Allocatable - 5Gi  
+/Bu/Pod3/m.limit = 3Gi  
+/Bu/Pod4/m.limit = 2Gi  
+```
+
+#### Best Effort
+
+For pods belonging to the Best effort Qos we don't set any quota.
+
+```
+/BE/cpu.shares = 2  
+/BE/cpu.quota= not set  
+/BE/m.limit_in_bytes = Allocatable - summation of memory requests of all Guaranteed and Burstable pods  
+/BE/PodUID/m.limit = no limit  
+```
+
+Example:
+We have a pod 'Pod5' having Pod Spec given below:
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod5
+spec:
+    containers:
+        name: foo
+            resources:
+        name: bar
+            resources:
+```
+
+Pod5 is classified as Best effort and is hence nested under the BE cgroup
+
+```
+/BE/cpu.shares = 2  
+/BE/cpu.quota= not set  
+/BE/m.limit_in_bytes = Allocatable - 7Gi  
+/BE/Pod5/m.limit = no limit  
+```
+
+### With Systemd
+
+In systemd we have slices for the three top level QoS class. Further each pod is a subslice of exactly one of the three QoS slices. Each container in a pod belongs to a scope nested under the qosclass-pod slice.
+
+Example:  We plan to have the following cgroup hierarchy on systemd systems
+
+```
+/memory/G-PodUID.slice/containerUID.scope
+/cpu,cpuacct/G-PodUID.slice/containerUID.scope
+/memory/Bu.slice/Bu-PodUID.slice/containerUID.scope
+/cpu,cpuacct/Bu.slice/Bu-PodUID.slice/containerUID.scope
+/memory/BE.slice/BE-PodUID.slice/containerUID.scope
+/cpu,cpuacct/BE.slice/BE-PodUID.slice/containerUID.scope
+```
+
+### Hierarchy Outline
+
+- "$Root" is the system root of the node ie. "/" by default and if `--cgroup-root` is specified then the specified cgroup-root is used as "$Root".
+- We have a top level Qos cgroup for the `Bu`(Burstable) and `BE`(Best Effort) Qos classes.
```

Comment:
/s/Qos/QoS

I imagine by this point `Bu` is understood in the proposal and you don't need to redefine it (same for others)